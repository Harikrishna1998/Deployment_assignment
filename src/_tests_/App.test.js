import React from "react";
import App from "../App";
import { shallow,configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });


describe("Content section snapshot test ", () => {
  it("the snapshot should match", () => {
    const container = shallow(<App />);
    expect(container).toMatchSnapshot();
  });
});
