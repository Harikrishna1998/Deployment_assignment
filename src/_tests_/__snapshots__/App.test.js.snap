// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content section snapshot test  the snapshot should match 1`] = `ShallowWrapper {}`;
